e2etests/cypress/support/e2e.js

```diff
@@ -0,0 +1,10 @@
+import './commands'
```

Comment:
This file can be removed , as  we will be adding ssl cert

Reply:
This is a placeholder as part of the setup. Can remove this if its not required as part of entire implementation of test scripts for PIRO